_posts/en/newsletters/2020-04-08-newsletter.md

```diff
@@ -73,30 +73,28 @@ question below to see a summary of the answer from the meeting._
 
 [Retry notfounds with more urgency][review club 18238] is a PR
 ([#18238][Bitcoin Core #18238]) by Anthony Towns that would change peer-to-peer
-behavior so that when nodes receive a `notfound` in response to a request for a
+behavior so that when nodes receive a `NOTFOUND` message in response to a request for a
 transaction, they would skip the current time-out period and instead try
 to obtain the transaction as fast as possible from another peer.
 
 Discussion began with fundamental reasons for the PR:
 
 <div class="review-club-questions"></div>
-- <details><summary>Why could retrying <code>notfound</code> more quickly be helpful?</summary>
+- <details><summary>Why could retrying <code>NOTFOUND</code> more quickly be helpful?</summary>
   DoS prevention, transaction propagation speed, privacy, and future <code>mapRelay</code>
   removal.</details>
 - <details><summary>What is a potential DoS attack concern?</summary>
-  Nodes with small mempools could force peers to wait a long time before
-  receiving a transaction.</details>
+  Nodes with small mempools could inadvertently slow transaction relay to
+  peers by announcing a transaction and then not being able to deliver it.</details>
 - <details markdown="1"><summary>Why is transaction propagation speed important?</summary>
   Short delays in seconds aren't an issue (and can even be desirable for
-  privacy), but larger delays in minutes can hurt propagation of transactions,
-  relay of [BIP152][] compact blocks, and even that of blocks.
+  privacy), but larger delays in minutes can hurt propagation of transactions and
+  relay of [BIP152][] compact blocks.
   </details>
 - <details><summary>When and why was <code>mapRelay</code> originally added?</summary>
-  <code>mapRelay</code> was present in the first version of Bitcoin to ensure a
-  transaction could be downloaded even if a block had already confirmed it.</details>
-- <details><summary>Why could removing <code>mapRelay</code> be desirable?</summary>
-  <code>mapRelay</code> has poor memory bounding, which leads to memory exhaustion issues
-   that have blocked progress.</details>
+  <code>mapRelay</code> was present in the first version of Bitcoin. It ensures that if
+  a node announced a transaction, it can be downloaded even if it is confirmed in
+  a block between being announced and the peer requesting it.</details>
 - <details><summary>Describe one issue with removing <code>mapRelay</code>?</summary>
   It could cause requested transactions in honest situations to more often be
   <code>notfound</code> with delays of up to 2 minutes, hurting propagation.</details>
```

Comment:
suggest: `NOTFOUND` or "not found"

Reply:
FYI, I use lowercase `notfound` because the actual text string in the P2P messages is lowercase.

Reply:
Yes, `NOTFOUND` is the `NetMsgType` message name and "notfound" is the message content:
```cpp
src/protocol.cpp:33:const char *NOTFOUND="notfound";
```
 
Either way, would be good to use the same one throughout the section.

Reply:
You're right. Changed back to `notfound`. Thanks!